Scripts/AnalyzeDatabank/calcOrderParameters.py

```diff
@@ -73,7 +73,7 @@
          type(system['WARNINGS']) is dict and
          'GROMACS_VERSION' in system['WARNINGS'] and 
          system['WARNINGS']['GROMACS_VERSION'] == 'gromacs3' ):
-        trjconvCOMMAND = '/home/osollila/Programs/gromacs/gromacs402/bin/trjconv'
+        trjconvCOMMAND = 'trjconv'
```

Comment:
have you checked if this is working? Here we assume that if gmx3 is installed, trjconv is called via `trjconv`, not `gmx trjconv`

i'm not familiar with gmx3, hence the question

Reply:
I don't want to make unit-test covering gromacs3. I would prefer removing it completely. Do you want us to remove them in this PR?

Reply:
If it's not tested yet, I'd suggest we keep the old behavior for now and deal with it in the future

Reply:
> If it's not tested yet, I'd suggest we keep the old behavior for now and deal with it in the future
Issue is opened about it.